README.md

```diff
@@ -1,91 +1,101 @@
 # enchannel-zmq-backend
 
-![enchannel version](https://img.shields.io/badge/enchannel-1.1-ff69b4.svg)
+[![GitHub release](https://img.shields.io/badge/version-latest-blue.svg)](https://github.com/nteract/enchannel-zmq-backend/releases)
 
-The ZeroMQ backend for [`enchannel`](https://github.com/nteract/enchannel). The classic edition, since Jupyter relies on ZeroMQ first and foremost.
+[**Installation**](#installation) | [**Usage**](#usage) | 
+[**Contributors and developers**](#contributors-and-developers) | 
+[**Learn more about nteract**](#learn-more-about-nteract)
 
-## Installation
-
-`npm install enchannel-zmq-backend`
-
-### ZeroMQ Dependency
-
-For all systems, you'll need
-
-- [`npm`](https://docs.npmjs.com/getting-started/installing-node)
-- [ZeroMQ](http://zeromq.org/intro:get-the-software)
-- Python 2 (for builds - you can still run Python 3 code)
-
-Each operating system has their own instruction set. Please read on down to save yourself time.
+**enchannel-zmq-backend** offers the ZeroMQ backend implementation for 
+[`enchannel`](https://github.com/nteract/enchannel).
 
-#### OS X
+## Technical overview
 
-##### homebrew on OS X
+As a refresher for the reader, *enchannel* details nteract's
+lightweight, implementation-flexible specification for communication
+between a user frontend and a backend, such as a language kernel. The 
+*enchannel* specification offers a simple description of "what" 
+messages may be passed between frontends and backends, while leaving
+a developer freedom in "how" to achieve message communication.
 
-- [`pkg-config`](http://www.freedesktop.org/wiki/Software/pkg-config/): `brew install pkg-config`
-- [ZeroMQ](http://zeromq.org/intro:get-the-software): `brew install zeromq`
+**enchannel-zmq-backend** takes a classic design approach using ZeroMQ,
+the foundation messaging protocol for the Jupyter project.
+enchannel-zmq-backend implements backend support for the messaging
+channels described in the [Jupyter messaging specification][]. This 
```

Comment:
Probably want to link to http://jupyter-client.readthedocs.io/en/latest/messaging.html here.